benchmarks/benchmarks/common.py

```diff
@@ -25,40 +25,90 @@
     'complex256',
 ]
 
+
+def memoize(func):
+    result = []
+    def wrapper(*a, **kw):
+        if not result:
+            result.append(func(*a, **kw))
+        return result[0]
+    return wrapper
```

Comment:
Command line `cd benchmarks && asv continuous v1.10.1 master`.
If the release tag commits were benchmarked (they are not currently), you could clone the `numpy-bench` repository, `ln -s numpy-bench/results benchmarks/results` and use `asv compare` for the comparisons.